src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side, String color) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: " + String.format("%.1f", getArea())
```

Comment:
Instead of using 'String.format' for formatting the area, consider creating a constant for the format. This will make it easier to manage if the format needs to change and also makes the code more readable.